map/src/menu/MainMenu.js

```diff
@@ -164,6 +175,7 @@ export default function MainMenu({
     //open main menu if currentObjectType was changed
     useEffect(() => {
         if (ctx.currentObjectType) {
+            closeCloudSettings();
```

Comment:
Probably it is not so good idea to fetch list-files API every time when currentObjectType changed.

Please change approach and avoid requesting `list-files` every time.

For example, you can request list-files only when you have registed changes in the files (delete, upload, empty trash, restore, etc).

PS. Additionally, list-files requires login, but guest users should have ability to use "public accessible" features without login.

Reply:
I looked into it, which is why I implemented it as it is now. 

The regular 'list-files' used in tracks only retrieves tracks and favorites, and only the latest versions if not deleted.

In this case, however, all versions of all types of files are retrieved, and there are many more of them. Therefore, I implemented lazy loading, and that’s why after any actions, I do not make a redundant heavy request for all versions, but work with what is already there, updating the existing list. And that’s why the update of the old 'list-files' is made upon exiting the cloud menu. Without this update, it would not be possible.

Reply:
Let's test the final version. For the moment, I can not see list-files after any currentObjectType changed, anymore.